pkg/table-router/router_test.go

```diff
@@ -30,11 +30,11 @@ type testRouterSuite struct{}
 
 func (t *testRouterSuite) TestRoute(c *C) {
 	rules := []*TableRule{
-		{"Test_1_*", "abc*", "t1", "abc"},
-		{"test_1_*", "test*", "t2", "test"},
-		{"test_1_*", "", "test", ""},
-		{"test_2_*", "abc*", "t1", "abc"},
-		{"test_2_*", "test*", "t2", "test"},
+		{SchemaPattern: "Test_1_*", TablePattern: "abc*", TargetSchema: "t1", TargetTable: "abc"},
+		{SchemaPattern: "test_1_*", TablePattern: "test*", TargetSchema: "t2", TargetTable: "test"},
+		{SchemaPattern: "test_1_*", TablePattern: "", TargetSchema: "test", TargetTable: ""},
+		{SchemaPattern: "test_2_*", TablePattern: "abc*", TargetSchema: "t1", TargetTable: "abc"},
+		{SchemaPattern: "test_2_*", TablePattern: "test*", TargetSchema: "t2", TargetTable: "test"},
```

Comment:
please add some test for new added fields

Reply:
Hi @yufan022 it's good to add some example for new added fields in unit tests. Because for now I don't know the grammar of `TableRegexp`. Should it contain `()` to support capturing groups, or the whole regexp is to match a part of table names?

Reply:
Got it, I will refine the unit test later.